brainstormforce/ultimate-addons-for-gutenberg
# Fix: Section - Backward compatibility issue when switch from master to next-release.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/4d53bd0d-bb85-4209-9a43-7e3c0d0e4077?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix ( breaking change which fixes an issue )

### How has this been tested?

- Step to reproduce the issue.
  - Create a section in master with bg image and advance heading inside section block.
  - Updated and check front and editor.
  - Now Switch to next release and refresh the editor you will see the deprecated issue.
- Test case
  - Drag and drop section with the image as a background and some content inside a section block.
  - Switch to our fixed branch and check backward the issue is resolved or not.

### Checklist:
